libraries/AP_NavEKF2/AP_NavEKF2_PosVelFusion.cpp

```diff
@@ -168,18 +168,21 @@ void NavEKF2_core::SelectVelPosFusion()
     // Select height data to be fused from the available baro, range finder and GPS sources
     selectHeightForFusion();
 
-    // If we are operating without any aiding, fuse in the last known position and zero velocity
+    // If we are operating without any aiding, fuse in the last known position
     // to constrain tilt drift. This assumes a non-manoeuvring vehicle
     // Do this to coincide with the height fusion
     if (fuseHgtData && PV_AidingMode == AID_NONE) {
         gpsDataDelayed.vel.zero();
-        // only fuse synthetic measurements when rate of change of velocity is less than 1g to reduce attitude errors due to launch acceleration
-        if (accNavMag < 9.8f) {
-            fuseVelData = true;
+        gpsDataDelayed.pos.x = lastKnownPositionNE.x;
+        gpsDataDelayed.pos.y = lastKnownPositionNE.y;
+        // when in flight only fuse synthetic measurements when rate of change of velocity is less than 1.1g
+        // to reduce attitude errors due to launch acceleration
+        if (accNavMag < 1.1f * GRAVITY_MSS || motorsArmed) {
```

Comment:
Yes, thanks for picking that up. That was intended to be 
if (accNavMag < 1.1f * GRAVITY_MSS || !motorsArmed) {